docs/intro.md

```diff
@@ -11,7 +11,12 @@ slug: /
   <meta name="google-site-verification" content="jXAJxglm5PRXRtL0R8kfGf80Xs1WM3IVAwAUPJC23lI" />
 </head>
 
-Momento Cache is the world's first truly serverless caching service. It provides instant elasticity, scale-to-zero capability, and blazing-fast performance. Gone are the days where you need to choose, manage, and provision capacity. With Momento Cache, you grab the SDK, you get an end point, input a few lines into your code, and you're off and running.
+Momento provides instantly scalable and simply priced data platform you can take to production today. It currently provides
+the below suite of serverless distributed services:
+
+- Momento Cache is the world's first truly serverless caching service. It provides instant elasticity, scale-to-zero capability, zero infrastructure management, and blazing-fast performance. With Momento Cache, you grab the SDK, you get an end point, input a few lines into your code, and you're off and running. [Learn More](./introduction/what-is-serverless-caching.md)
```

Comment:
While we are changing things, we should get the newer text for this. "world's first truly serverless caching..." need to be removed.